src/NUnitEngine/nunit.engine.core/Services/ExtensionManager.cs

```diff
@@ -131,6 +135,24 @@ public void FindExtensions(string startDir)
                 FindExtensionsInAssembly(candidate);
         }
 
+        /// <summary>
+        /// Find and install standard extensions for a host assembly using
+        /// a built-in algorithm that searches in certain known locations.
+        /// </summary>
+        /// <param name="hostAssembly">An assembly that supports extensions.</param>
+        public void FindStandardExtensions(Assembly hostAssembly)
+        {
+            bool isChocolateyPackage = System.IO.File.Exists(Path.Combine(hostAssembly.Location, "VERIFICATION.txt"));
+            string[] extensionPatterns = isChocolateyPackage
+                ? new[] { "nunit-extension-*/**/tools/", "nunit-extension-*/**/tools/*/" }
+                : new[] { "NUnit.Extension.*/**/tools/", "NUnit.Extension.*/**/tools/*/" };
+
+            FindExtensionAssemblies(hostAssembly, extensionPatterns);
```

Comment:
Personally I prefer methods returning the set of found assemblies instead of setting a field. 
But the approach is pre-existing.